Editor/EditorCore/CutTool.cs

```diff
@@ -946,12 +946,16 @@ List<Face> ComputeNewFaces(Face face, IList<int> cutVertexIndexes)
 
                      if(index >= 0)
                      {
-                         List<Face> toDelete;
-                         Face newFace = ComputeFaceClosure(polygon, index, cutVertexSharedIndexes, out toDelete);
-                         newFace.submeshIndex = m_TargetFace.submeshIndex;
-
-                         newFaces.Add(newFace);
-                         facesToDelete.AddRange(toDelete);
+                         // In the case of only 2 distinct, a face should not be added.
+                         if (polygon.Distinct().Count() != 2)
```

Comment:
I think, that should be just:
```suggestion
                         if (polygon.count != 2)
```
Reason is we should minimize the use of Linq, and also they should already be distinct :D

Reply:
Thank you !